src/openforms/forms/api/public_api/filters.py

```diff
@@ -0,0 +1,10 @@
+from django_filters import rest_framework as filters
+
+from openforms.forms.models import Form
```

Comment:
```suggestion
from ...models import Form
```
consistent with the views :)